rmf_fleet_adapter_python/src/adapter.cpp

```diff
@@ -140,7 +140,28 @@ PYBIND11_MODULE(rmf_adapter, m) {
       return self.unstable().get_participant();
     },
     py::return_value_policy::reference_internal,
-    "Experimental API to access the schedule participant");
+    "Experimental API to access the schedule participant")
+  .def("set_unstable_lift_entry_watchdog",
+    [&](agv::RobotUpdateHandle& self,
+        agv::RobotUpdateHandle::Unstable::Watchdog watchdog,
+        rmf_traffic::Duration wait_duration)
+    {
+      self.unstable().set_lift_entry_watchdog(watchdog, wait_duration);
+    },
+    py::arg("watchdog"),
+    py::arg("wait_duration") = std::chrono::seconds(10),
+    "Experimental API to set the lift entry watchdog");
+
+  auto m_robot_update_handle = m.def_submodule("robot_update_handle");
+
+  py::enum_<agv::RobotUpdateHandle::Unstable::Decision>(
+    m_robot_update_handle, "Decision")
```

Comment:
Maybe we can explicitly mention this as `unstable`, or create an unstable  `def_submodule`.